.pre-commit-config.yaml

```diff
@@ -0,0 +1,24 @@
+repos:
```

Comment:
Its probably worth trying to sync with pandas on this, which uses one repo for most checks